### DISCUSSION TITLE
Import image sequence

### DISCUSSION THREAD

**Discussion by Robiulkarimminar** [keiwando/toonsquid#43] [Q&A]
Is there any plan to add " import image sequence " in the upcoming update?

**Comment by keiwando** [comment]
Hi! It's not yet planned for any specific update but I'm keeping an eye on how often it gets requested in order to prioritize it accordingly.

**Reply by wandrinsheep** [reply]
Bump for this, I needed this exact feature moving some animation over from Callipeg was a pain

**Reply by keiwando** [reply]
This feature is now available with the 1.7 update. Thanks for your feedback!

**Reply by wandrinsheep** [reply]
you're amazing

**Comment by mshoek** [comment]
I would definitely make use of it!

**Comment by keiwando** [comment]
I managed to squeeze this feature into the 1.7 update. Let me know if you run into any issues with it.

**Reply by mshoek** [reply]
Wow! It works perfectly. Thank you!
